modules/jdbc-test/src/test/java/org/testcontainers/jdbc/DatabaseDriverTest.java

```diff
@@ -0,0 +1,50 @@
+package org.testcontainers.jdbc;
+
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.testcontainers.containers.JdbcDatabaseContainer;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+import static org.rnorth.visibleassertions.VisibleAssertions.assertNotNull;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertNull;
+import static org.rnorth.visibleassertions.VisibleAssertions.assertTrue;
+
+/**
+ * Created by inikolaev on 08/06/2017.
+ */
+public class DatabaseDriverTest {
```

Comment:
Please could you rename to be slightly more specific? i.e., this could be called `DatabaseDriverShutdownTest` or something like that.